### DISCUSSION TITLE
Disable feature preview notifications

### DISCUSSION THREAD

**Discussion by keanerickson-mc** [community/community#47000] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, is there a way to permanently disable the notification dot, relating to feature previews?
Or will I just have to ignore it.

**Comment by Hangell** [comment]
Infelizmente, não existe uma maneira de desativar permanentemente o ponto de notificação relacionado às visualizações de recursos no GitHub. A única maneira de lidar com isso é ignorá-lo. Você pode criar uma regra no seu email e mover esses emails para uma pasta e uma regra para excluir depois que cair nessa pasta.

**Reply by Hangell** [reply]
> Infelizmente, não existe uma maneira de desativar permanentemente o ponto de notificação relacionado às visualizações de recursos no GitHub. A única maneira de lidar com isso é ignorá-lo. Você pode criar uma regra no seu email e mover esses emails para uma pasta e uma regra para excluir depois que cair nessa pasta.

Para criar a regra é só seguir os passos abaixo e vai conseguir sem erro!

Abra o Microsoft Outlook e clique no botão "Arquivo" na barra de ferramentas.

Selecione "Opções" na seção "Informações".

Selecione "Regras e Alertas" na seção "Configurações da Caixa de correio".

Clique no botão "Nova regra" na seção "Regras".

Selecione a opção "Mover mensagens com base em determinadas condições" e clique em "Próximo".

Selecione a opção "Com as palavras no campo Assunto" e digite "github" na caixa de texto.

Clique no botão "Próximo" e selecione a opção "Mover para a pasta".

Selecione a pasta desejada na lista de pastas ou crie uma nova pasta clicando no botão "Nova pasta".

Clique no botão "Finalizar" para concluir a criação da regra.

Agora, todos os e-mails do GitHub que chegarem à sua caixa de entrada serão automaticamente movidos para a pasta selecionada de acordo com a regra criada.

**Comment by haadr** [comment]
For those who get anxious about the constant notification dot, here's a ublock origin filter to remove it.

```
github.com##.Header-link > .js-feature-preview-indicator.unread-indicator
```
